backend/mobilityAI/Controllers/SensorDataController.cs

```diff
@@ -108,87 +108,37 @@ public async Task<IActionResult> AddSensorData(int patientId, IFormFile accelero
         }
 
         /// <summary>
-        /// Add a single file with accelerometer and gyroscope data into the database and into machine learning server
-        /// TODO: Handle larger files (ie: don't wait for the files to be added to the database before returning 200 or else the client will timeout)
+        /// Adds or updates a step count for a patient for a given date
         /// </summary>
-        /// <param name="dataFile">
-        /// The file input for the data to be analyzed
-        /// </param>
-        /// <param name="patientID">
-        /// The patient id from which this data came from
-        /// </param>
-        [HttpPost("{patientId}")]
-        public async Task<IActionResult> AddSensorDataSingle(int patientId, IFormFile dataFile)
-        {
-            List<String> accel = new List<string>();
-            List<String> gyro = new List<string>();
-            string ac = "epoch (ms),time (-13:00),elapsed (s),x-axis (g),y-axis (g),z-axis (g)" + Environment.NewLine;
-            string gy = "epoch (ms),time (-13:00),elapsed (s),x-axis (deg/s),y-axis (deg/s),z-axis (deg/s)" + Environment.NewLine;
-
-            using (var reader = new StreamReader(dataFile.OpenReadStream()))
-            {
-                while (reader.Peek() >= 0)
+        /// <param name="patientId">ID of patient</param>
+        /// <param name="steps">Number of steps taken on given date</param>
+        /// <param name="date">Date the step count was recorded</param>
+        /// <returns>Status 200 if successful</returns>
+        [HttpPost("AddStepCount")]
```

Comment:
You should use 
```C#
[HttpPost("{patientId}/StepCount")]
public IActionResult AddStepCount(int patientId, int steps, String date)
```

so that when we have `GetStepCount` we can do

```C#
[HttpGet("{patientId}/StepCount")]
public IActionResult AddStepCount(int patientId, String date)
```